toml/src/main/resources/META-INF/toml-only.xml

```diff
@@ -32,6 +32,16 @@
                          enabledByDefault="true" level="WARNING"
                          implementationClass="org.rust.toml.inspections.CrateNotFoundInspection"/>
 
+        <intentionAction>
+            <className>org.rust.toml.intentions.ExpandDependencySpecificationIntention</className>
+            <category>Rust</category>
```

Comment:
Let's extract these new intentions into separate subgroup like `Cargo.toml`. Note, I'm note sure about category syntax to create a subgroup but you can check platform inspections